pandas/io/json/__init__.py

```diff
@@ -1,12 +1,13 @@
 from pandas.io.json._json import dumps, loads, read_json, to_json
-from pandas.io.json._normalize import json_normalize
+from pandas.io.json._normalize import _json_normalize, json_normalize
 from pandas.io.json._table_schema import build_table_schema
 
 __all__ = [
     "dumps",
     "loads",
     "read_json",
     "to_json",
+    "_json_normalize",
```

Comment:
Does this need to be a part of `__all__`?

Reply:
Yes, because we are aliasing it in `pandas/__init__.py` as `json_normalize`.